### ISSUE TITLE
Fishing skill gains are set wrong

### ISSUE THREAD

**Issue by pottedfrog** [AbyssWoW/BugTracker#601]
The number of fish required to get 1 skill point are double what they should be on average.

I use the term 'on average' a lot - there _should_ be some variance, but I only noticed a variance of +/-1 - it basically means what has been seen over many hundreds of thousands of catches.

The table here shows the 'averages' required, which was produced from ElAngling (now defunct), but they tested using hundreds of thousands of catches from a lot of people, so the table is pretty accurate : https://wow.gamepedia.com/Fishing#Increasing_fishing_skill

In the archived version of ElsAngling this page also states is should take about 1200 fish to get from 1-700 : http://web.archive.org/web/20150209134315/http://www.elsanglin.com/role_of_skill.html

Currently I've seen at least double these rates at every single skill 'tier' above 75 skill - at 350+ it's taking 10-12 fish catches per skill level - which is way outside the normal variance. The least number of catches I've had to gain a skill level has been 10 since I hit 300 skill - again very much outside normal.

With almost 1200 fish caught, I should be closer to 450/460 skill - at the time of writing this I am at 365.

In conclusion I believe the current fishing system is using the Wrath 'average fish per gain' tables which changed once Cata was released.

IGN : Slimline
